src/chrome/content/monitor.js

```diff
@@ -252,10 +252,33 @@ Monitor.prototype.isHTML = function (doc) {
     is_html = (contentType == 'text/html' ||
                contentType == 'text/xhtml' ||
                contentType == 'application/xhtml+xml');
+
+    var blacklisted_domains = itsalltext.preferences.blacklist_domains.split("\n");
+    var pattern,
+        domain_is_blacklisted = false;
+    for(var i=0, len=blacklisted_domains.length; i<len; i++)
+    {
+        pattern = new RegExp( blacklisted_domains[i] );
+        if( pattern.test( doc.domain ) )
+        {
+            domain_is_blacklisted = true;
+            break;
+        }
+    }
+
+    /*
+    var console = Cc["@mozilla.org/consoleservice;1"].getService(Ci.nsIConsoleService);
```

Comment:
You can use the built-in IAT debugging stuff... there are examples around here someplace.